### PR TITLE
EN: Training tutorial, Even dialogue, some organisation

### PR DESCRIPTION
**Pull request by DabirSA**

### CODE REVIEW

**Comment by Aida-Enna** [comment]
Just ignore the travis CI error for now, sorry!

**Comment by alama** [comment]
> Just ignore the travis CI error for now, sorry!

Merge with upstream's EN for Travis CI fixes

File: Orders/Afin/co01020022.csv [hunk 1]
```diff
@@ -1,3 +1,3 @@
 Title#0,"""Setting Photon Arts?"""
-Explanation#0,"""よう、相棒！<br>フォトンアーツのセッティングって<br>ちゃんとやってるか？<br>武器パレットに設定しておかないと<br>習得していても使えないんだぜ。<br>ま、かくいうおれも勘違いしてて<br>同じのしか使ってなかったんだけどな。<br>お前ならそんなことはないと思うけど<br>復習って意味で、フォトンアーツの<br>セッティング、やってみたらどうだ？"""
+Explanation#0,"""Yo, partner!<br>Are you setting your Photon Arts?<br>Even if you've learned a Photon Art, youwon't be able to use it if you don't set it<br>on your Weapon Palette.<br>I made that mistake myself, and I ended<br>up using the same thing over and over.<br>I'm not saying that happened to you, but<br>could we go over how to do it anyway,<br>just as a kind of review?"""
```
Comment (alama) [inline Orders/Afin/co01020022.csv]: "youwon't" or "you won't"?

Comment (DabirSA) [reply Orders/Afin/co01020022.csv]: accidentally dropped a line break

**Pull request merged**